packages/contracts-periphery/contracts/universal/op-erc721/OptimismMintableERC721.sol

```diff
@@ -78,8 +85,8 @@ contract OptimismMintableERC721 is ERC721Enumerable, IOptimismMintableERC721 {
     /**
      * @inheritdoc IOptimismMintableERC721
      */
-    function mint(address _to, uint256 _tokenId) external virtual onlyBridge {
-        _mint(_to, _tokenId);
+    function safeMint(address _to, uint256 _tokenId) external virtual onlyBridge {
+        _safeMint(_to, _tokenId);
```

Comment:
Comment only: This introduces an external call from OZ's `_checkOnERC721Received`. Since the function is external, it should be fine.